src/context/WalletProvider/NativeWallet/components/NativeTestPhrase.tsx

```diff
@@ -27,13 +27,20 @@ type TestState = {
 
 export const NativeTestPhrase = ({ history, location }: NativeSetupProps) => {
   const [testState, setTestState] = useState<TestState | null>(null)
+  const [hasAlreadySaved, setHasAlreadySaved] = useState(false)
   const [invalidTries, setInvalidTries] = useState<number[]>([])
   const [testCount, setTestCount] = useState<number>(0)
   const [revoker] = useState(new (Revocable(class {}))())
   const [shuffledNumbers] = useState(slice(shuffle(range(12)), 0, TEST_COUNT_REQUIRED))
   const [, setError] = useState<string | null>(null)
```

Comment:
why do we have setError without error?

Reply:
Does it block this PR? Maybe we could open a dedicated issue in order to implement error handling there